example/src/VideoroomContext.tsx

```diff
@@ -0,0 +1,39 @@
+import { SERVER_URL } from '@env';
+import * as React from 'react';
+import { useState, Dispatch, SetStateAction } from 'react';
+import { Platform } from 'react-native';
+
+type VideoroomContextType = {
+  roomNameState: [string, Dispatch<SetStateAction<string>>];
+  serverUrlState: [string, Dispatch<SetStateAction<string>>];
+  displayNameState: [string, Dispatch<SetStateAction<string>>];
+  isSimulcastOnState: [boolean, Dispatch<SetStateAction<boolean>>];
+};
+
+const VideoroomContext = React.createContext<VideoroomContextType | undefined>(
+  undefined
+);
+
+const VideoroomContextProvider = (props) => {
+  const [roomName, setRoomName] = useState<string>('room');
+  const [serverUrl, setServerUrl] = useState<string>(SERVER_URL);
+  const [displayName, setDisplayName] = useState<string>(
+    `mobile ${Platform.OS}`
+  );
+  const [isSimulcastOn, setIsSimulcastOn] = useState<boolean>(true);
```

Comment:
Do we even need this in the context? We're not using it anywhere besides Home screen. This applies also to `serverUrl`. In the context we should only store things that are needed globally in the app.

Reply:
Moved to `Home.tsx`